tests/steps/workflowSubmitArticle.js

```diff
@@ -0,0 +1,30 @@
+let counter = 0;
+
+module.exports = () => {
+  counter++;
+
+ return {
+    [`[${counter}] submit the article via the "Workflow" menu in the dialog box`]: (client) => {
+      const modalBlogSelector = '.apostrophe-blog-manager';
+      const modalEditArticleSelector = '.apos-pieces-editor';
+      const manageTableRowSelector = '.apos-manage-table tr[data-piece]';
+      const editArticleBtnSelector = `${manageTableRowSelector} a`;
+      const controlSelector = `${modalEditArticleSelector} .apos-modal-controls .apos-dropdown`;
+      const workflowModalBtnSelector =
+        `${modalEditArticleSelector} [data-apos-dropdown-name="workflow"]`;
+      const submitWorkflowBtnSelector = `${modalEditArticleSelector} [data-apos-workflow-submit]`;
+
+      client.waitForElementVisible(modalBlogSelector);
+      client.waitForElementVisible(manageTableRowSelector);
+      client.click(editArticleBtnSelector);
+      client.waitForElementVisible(modalEditArticleSelector);
+      client.waitForElementVisible(controlSelector);
+      client.pause(1000);
```

Comment:
A lot of pause() going on, I feel like this needs to go. Talk to me about why it's necessary. There could be issues where I'm waiting for DOMready to do stuff or just doing things that are too fast for human users to be bothered by but too slow for you?

Reply:
This was leftover from debugging. Removed.